packages/letspeppol-sdk-php/src/Resources/ProxyClient.php

```
@@ -0,0 +1,328 @@
+     * @return array Updated document with new schedule
+     * @throws ApiException When document not found (404) or already sent (409)
+     */
+    public function rescheduleDocument(string $id, array $documentData): array
+    {
+        return $this->put("/sapi/document/{$id}/send", $documentData);
+    }
+
+    /**
+     * Mark document as downloaded
+     */
+    public function markDownloaded(string $id, bool $noArchive = false): void
```

PHPMD (2.15.0)

The method markDownloaded has a boolean flag argument $noArchive, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)